source/clear-linux/tutorials/wp-install/wp-install.rst

```diff
@@ -1,215 +1,170 @@
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo curl -O https://wordpress.org/latest.tar.gz
 
-3. To extract all the files and directories from the downloaded file,
-   enter the following command:
+#. Extract all files and directories from the downloaded file:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo tar –xzvf latest.tar.gz
 
-4. To rename the top-level WordPress directory to “blog”, enter the following
-   command:
+#. Rename the top-level WordPress directory to “blog”:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo mv wordpress blog
 
-5. To remove the downloaded tar file, enter the following command:
+#. Remove the downloaded tar file:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo rm latest.tar.gz
 
-Setting up WordPress
---------------------
+Set up WordPress with Web-based GUI
+-----------------------------------
 
-With the WordPress components loaded into the
-:file:`/var/www/html/blog` directory, we can set everything up.
+Recall that you created a database and user when you installed phpMyAdmin during :ref:`Creating a Clear Linux based web server <web-server-install>`. Next, you must connect WordPress to the database and install WordPress.
 
-Instead of editing the :file:`wp_config.php` file manually, we are using the
-web-based configuration tool to setup the database name and user. We created
-both in the :ref:`web server tutorial <web-server-install>`. In your browser,
-go to: http://<your_website>/blog/wp-admin/install.php.
+To continue with the setup, go to: http://<your_website>/blog/wp-admin/install.php. The WordPress language option screen appears, as shown in figure 1.
 
-Your screen should look like figure 1:
+#. Select :guilabel:`English` and click :guilabel:`Continue`.
 
-.. _fig-wp-install-1:
-
-.. figure:: figures/wp-install-1.png
+   .. figure:: figures/wp-install-1.png
     :alt: WordPress language selection
     :width:     600
 
-    WordPress language selection screen.
-
-Select :guilabel:`English` for the language and click the
-:guilabel:`Continue` button. :ref:`As shown on figure 1<fig-wp-install-1>`
+    `Figure 1: WordPress language selection screen.`
 
 
+   The WordPress installation continues until the Welcome screen appears, as shown in figure 2:
 
-The WordPress installation continues until the welcome screen shown in figure
-2 appears:
-
-.. figure:: figures/wp-install-2.png
+   .. figure:: figures/wp-install-2.png
     :alt: WordPress welcome screen
     :width:     600
 
-    WordPress welcome screen.
-
-Click the :guilabel:`Let’s go!` button to enter the information.
+    `Figure 2: WordPress Welcome screen.`
 
-Enter the database name, username, and password we used when creating the
-database:
+#. Click :guilabel:`Let’s go!`.
+#. Enter database connection specifics in the screen that appears as shown in Figure 3 below.
 
-   Database name:       WordPress
+   - Database name:       WordPress
```

Comment:
Just FYI: Bullets below are acceptable. 
In contrast, the CL team generally uses '*' for bullets. So going forward, please use the latter style.